src/citrine/resources/file_link.py

```diff
@@ -192,14 +193,44 @@ def __str__(self):
 class FileCollection(Collection[FileLink]):
     """Represents the collection of all file links associated with a dataset."""
 
-    _path_template = 'projects/{project_id}/datasets/{dataset_id}/files'
+    _path_template = 'teams/{team_id}/datasets/{dataset_id}/files'
```

Comment:
Will the `projects` version of the endpoint be dead?

Reply:
We'll kill it after a while, but given that we are using the datasetId we can freely use the teams-based one for both project->dataset and team->dataset scenarios. It is going to return exactly the same data.